typescript/cli/src/config/warp.ts

```diff
@@ -134,15 +140,15 @@ export async function createWarpRouteDeployConfig({
     const isNft =
       type === TokenType.syntheticUri || type === TokenType.collateralUri;
 
-    const mailbox = await detectAndConfirmOrPrompt(
-      async () => {
-        const addresses = await context.registry.getChainAddresses(chain);
-        return addresses?.mailbox;
-      },
-      `For ${chain}, enter the`,
-      'mailbox address',
-      'hyperlane-registry',
-    );
+    // default to the mailbox from the registry and if not found ask to the user to submit one
```

Comment:
Imho, this prompt should be moved up to right after the chain selection. This is because the prompt depends on the chain selection, and users should not have to go back and resolve chain-related prompts after something else (select token type). 

1. select chain
2. select token type
3. Prompt for mailbox address because chain does not have it.

I think it should be
1. select chain
2. Prompt for mailbox address because chain does not have it.
3. select token type

Reply:
I agree, it makes more sense to have it there thanks for the suggestion!